middleware/src/handlers/handlers.go

```diff
@@ -2,18 +2,21 @@
 package handlers
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
 	"net/http"
 
 	middleware "github.com/digitalbitbox/bitbox-base/middleware/src"
+	"github.com/digitalbitbox/bitbox-base/middleware/src/system"
 	"github.com/gorilla/mux"
 	"github.com/gorilla/websocket"
 )
 
 // Middleware provides an interface to the middleware package.
 type Middleware interface {
 	Start() <-chan *middleware.SampleInfo
+	GetSystemEnv() system.Environment
```

Comment:
(Optional) Could we add brief comments on what `Start()` and `GetSystemEnv()` returns in this interface?

Reply:
Done.